### DISCUSSION TITLE
Grammar error on wiki.js landing page

### DISCUSSION THREAD

**Discussion by mammabear123** [requarks/wiki#7045] [Error / Bug Report]
Nothing major, just a grammatical error in the Scalable section of the English [https://js.wiki/](https://js.wiki/) page - underline is mine.

## Scalable
Whether it's on a tiny Raspberry Pi or on a high performance VM in the cloud, Wiki.js __intelligently make use of__ the available resources.

Some grammatically correct alternatives to the underlined text:
- will intelligently make use of
- intelligently makes use of
- makes intelligent use of

I hope this is considered more helpful than annoying. :)

**Comment by NGPixel** [comment]
Fixed. Thanks!
